openslides_backend/action/motion/create.py

```diff
@@ -1,7 +1,53 @@
+from typing import Any, Dict
+
+from ...models.models import Motion
+from ...shared.exceptions import ActionException
+from ...shared.patterns import Collection, FullQualifiedId
 from ..action import register_action
-from ..base import DummyAction
+from ..default_schema import DefaultSchema
+from ..generics import CreateAction
 
 
 @register_action("motion.create")
-class MotionCreate(DummyAction):
-    pass
+class MotionCreate(CreateAction):
```

Comment:
Please subclass from `CreateActionWithAgendaItem` (or whatever I called it) and add tests for agenda item creation.

Reply:
Added the CreateActionWithAgendaItemMix here and add a test